app/models/user_session.rb

```diff
@@ -56,6 +54,19 @@ def user_agent=(value)
     self[:user_agent] = value
   end
 
+  private_class_method def self.user_session_ttl_parse(duration)
```

Comment:
By the way you should just put this method one block below and you do not have to use `private_class_method`

Reply:
I moved it. With `private_class_method` I wanted to make clear it is an internal utility method not for reuse. And I have to leave it, otherwise robocop gives a warning

```
private (on line 57) does not make singleton methods private. Use private_class_method or private inside a class << self block instead.
```

Moving to a utility class IMO better be done once your suggestion above for a new configuration class is implemented but let me know if you want to see it elsewhere.

Please review again. Hope no other validator will find an issue with it now.

Reply:
I was thinking of that yes, sorry I did not make myself clear enough

private

class << self
  def user_session_ttl_parse(duration)
    # ...
  end
end